components/admin_console/team_channel_settings/team/list/team_list.tsx

```diff
@@ -228,6 +228,15 @@ export default class TeamList extends React.PureComponent<Props, State> {
                             <div className='TeamList_nameText'>
                                 <b data-testid='team-display-name'>
                                     {team.display_name}
+                                    {team.delete_at !== 0 &&
+                                        <span className='archived-label'>
```

Comment:
Can we add parens here as well?

```suggestion
                                    {team.delete_at !== 0 && (
                                        <span className='archived-label'>
```